Makefile.js

```diff
@@ -773,9 +773,10 @@ target.checkRuleFiles = function() {
          */
         function hasIdInTitle(id) {
             var docText = cat(docFilename);
-            var idInTitleRegExp = new RegExp("^# (.*?) \\(" + id + "\\)");
+            var idAtEndOfTitleRegExp = new RegExp("^# (.*?) \\(" + id + "\\)"); // original format
+            var idInTitleRegExp = new RegExp("^# " + id + ": "); // new format is same as rules index
```

Comment:
Just in terms of nomenclature, I think this variable could be named `idAtBeginningOfTitleRegExp` and (though it is longer) it would be more readable. Semantically, an ID at the end of a title is also an ID in a title, so I find the current names a little confusing in that they imply non-mutual exclusiveness but in reality are mutually exclusive.